CourseApp.Tests/TaskTest.cs

```diff
@@ -0,0 +1,19 @@
+namespace CourseApp.Tests
+{
+    using Xunit;
+
+    public class TaskTest
+    {
+        [Theory]
+        [InlineData(2, 1.2, 0.025)]
+        [InlineData(2, 1.8, 0.023)]
+        [InlineData(2, 2.4, 0.021)]
+        [InlineData(2, 3, 0.02)]
+        [InlineData(2, 3.6, 0.018)]
+        public void TestCalc(double a, double x, double expected)
+        {
+            var actual = new CalculateTest().Calculate(a, x);
+            Assert.Equal(expected, actual, 3);
+        }
```

Comment:
Please add tests for taskA and taskB

There you may check only size of returned array for cases:

**taskA**
- Xn < Xk, Xn +Dx < Xk
- Xn > Xk
- Xn + Dx > Xk

**taskB**
- array with n elements passed
- empty array passed